src/reply.rs

```diff
@@ -193,15 +198,43 @@ where
 
 /// Types that can be converted into a `Response`.
 ///
-/// This trait is sealed for now (implementations are only allowed inside
-/// warp), but it is implemented for the following:
+/// This trait is implemented for the following:
 ///
 /// - `http::StatusCode`
 /// - `http::Response<impl Into<hyper::Body>>`
 /// - `String`
 /// - `&'static str`
+///
+/// # Example
+///
+/// ```rust
+/// use warp::{Filter, http::Response};
+///
+/// struct Message {
+///     msg: String
+/// }
+///
+/// impl warp::Reply for Message {
+///     fn into_response(self) -> warp::reply::Response {
+///         Response::builder()
```

Comment:
This can skip the builder and unwrap: `Response::new(format!("message: {}", self.msg).into())`.